lib/sidebar-manager.js

```diff
@@ -19,6 +19,9 @@ onSidebarEvent('action', function(action) {
         case 'hideSidebar':
             module.exports.sidebarManager.content.hide();
             break;
+        case 'incrementStep':
+            module.exports.sidebarManager.setSidebarProps();
```

Comment:
Lots of `module.exports` again, but if it's too much of a pain to refactor please ignore :)